modules/admin-ui/src/main/webapp/scripts/app.js

```diff
@@ -106,9 +106,14 @@ angular.module('adminNg', [
         };
     }]);
 }])
-.config(function(hotkeysProvider) {
-    hotkeysProvider.includeCheatSheet = false;
-})
+.config(['hotkeysProvider', function (hotkeysProvider) {
+    hotkeysProvider.includeCheatSheet = true;
+    hotkeysProvider.templateTitle = 'HOTKEYS.CHEAT_SHEET.TITLE';
+    hotkeysProvider.cheatSheetDescription = 'HOTKEYS.KEYS.CHEAT_SHEET';
+    hotkeysProvider.template = hotkeysProvider.template
```

Comment:
It might be work to think about whether it wouldn't make more sense to provide a custom template here. That would avoid the "replace" that depend on implementation details of a library, the need of providing a vendor CSS and would enable the cheatsheet to fit within the Admin UI design.

Reply:
I used `replace` here because I did not want to copy the template which we would then have to maintain and keep in sync with the original one. However, if you say we want a custom one anyway, that is a moot point. And I would even agree with that; I just wanted to see first whether this is something the community wants at all, before diving into extensive design work, which I am admittedly not the best at. :wink:

Reply:
Actually you do not need to dive into design work at all: The Admin UI comes with all the required styles - just re-use them.